example/README.md

```diff
@@ -0,0 +1,30 @@
+# Kompari Xtask example
+
+This is an example that demonstrates how to integrate `kompari` as xtask into a test.
+
+* `demolib` is an example library that we are testing with snapshot tests
+* `xtask-kompari` is xtask that integrates kompari into the example project
+* `tests` is a directory with snapshots and image for the current tests
+
+
+## Demo
+
+Try to modify the test in `demolib`:
+
+E.g. changing the test in the following way:
+
+```rust
+    #[test]
+    fn test_create_rectangle() {
+        //      Change the value ----\
+        //      here to e.g. 25      |
+        //                           v
```

Comment:
I'd suggest moving this entire code block to inside the actual code. It took me a couple of readings to realise you didn't mean edit in the README, but in the same code inside the folder.

Reply:
When I say the code block, I mean the comments.

And I agree that having a practical demo is really good, and so if implementing this, we would point towards the moved comment in the README.